firefox-ios/Client/Frontend/Browser/Toolbars/AddressToolbarContainer.swift

```diff
@@ -23,13 +23,34 @@ class AddressToolbarContainer: UIView,
 
     private var windowUUID: WindowUUID?
     private var profile: Profile?
-    private var toolbarState: ToolbarState?
     private var model: AddressToolbarContainerModel?
     private var delegate: AddressToolbarContainerDelegate?
 
+    private var toolbarType: BrowserAddressToolbar {
```

Comment:
Maybe rename this to just `toolbar` or `displayedToolbar`? From the name `toolbarType` I would expect a value of `compact` or `normal` but not the displayed toolbar.